src/common/lib/api.ts

```diff
@@ -70,26 +71,37 @@ export const swrGetAccountInfo = async (
   const accountsCache = await getAccountsCache();
 
   return new Promise((resolve, reject) => {
-    if (accountsCache[id]) {
+    // don't apply cache for connectors that failed to connect
+    if (accountsCache[id] && !accountsCache[id].error) {
```

Comment:
Can we keep accounts with errors out of the cache? Why would we cache them if we exclude them from the cache here? :thinking:

Reply:
I changed it.